docs/conf.py

```diff
@@ -0,0 +1,48 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+"""
+Configuration file for mewbot sphinx document build.
+"""
```

Comment:
The comments here feel like they should be part of the doc block.

Reply:
Agreed - corrected